float64 height
float64 radius
std_msgs/ColorRGBA color
---
# Empty